device-registry/src/main/scala/org/genivi/sota/device_registry/GroupInfo.scala

```diff
@@ -31,10 +33,23 @@ object GroupInfo extends SlickJsonHelper {
 
   val groupInfos = TableQuery[GroupInfoTable]
 
-  protected def getGroup(groupName: String, namespace: Namespace) =
+  case class ValidName()
+
+  type Name = Refined[String, ValidName]
+
+  implicit val validGroupName: Validate.Plain[String, ValidName] =
```

Comment:
It seemed sensible to put some constraints on what a groupName can be. I wanted to extract groupName from inside the url path, to make it consistent with the system_info routes.